arangodantic/operators.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+
+
+class Operators(str, Enum):
+    """
+    Enum class for different operators.
+
+    See:
+    https://www.arangodb.com/docs/stable/aql/operators.html#comparison-operators
+    https://www.arangodb.com/docs/stable/aql/operators.html#array-comparison-operators
+    for more details.
+    """
+
+    LT = "<"
+    LTE = "<="
+    GT = ">"
+    GTE = ">="
+    NE = "!="
+    EQ = "=="
+    IN = "IN"
+    NOT_IN = "NOT IN"
+    ALL_IN = "ALL IN"
+    NONE_IN = "NONE IN"
+    ANY_IN = "ANY IN"
+    ANY = "ANY"
+    ALL = "ALL"
+    NONE = "NONE"
```

Comment:
It seems like there's no test case using the `ANY_IN`, `ANY`, `ALL` and `NONE`. Could you please add tests that would use these as well?

Based on the examples at https://www.arangodb.com/docs/stable/aql/operators.html I'm actually wondering if one is even supposed to be able use `ANY`, `ALL` and `NONE` as a stand alone operator; it looks to me more like they're always followed by another operator, like `ANY ==` or `ANY >` and I'm unsure if the logic building queries will actually work with that or not. I guess you'll find out when adding the tests. Those tests will also then be good examples on how to actually use those operators.

Reply:
@joakimnordling You're absolutely right. The Array Comparison operators require the other operand as well. I solved it by adding the different variations of them to the supported operators. Also added the same test cases as found in ArangoDB docs.